deploy/kubernetes/helm/templates/tools.yaml

```diff
@@ -145,8 +145,8 @@ spec:
               -D heron.uploader.s3.region={{ .Values.uploader.s3Region }}
               {{- end }}
               {{- if eq .Values.packing "RoundRobin" }}
-              -D heron.class.packing.algorithm=org.apache.heron.packing.roundrobin.RoundRobinPacking
```

Comment:
Ahh.  This needs to be changed back to `org.apche`, but I'm guessing this was done since they were using `0.17.8`.  We have  a little work to support the newer containers. Good catch @nicknezis.  I'm on the fence about this since we don't have an official Apache Docker container yet.  Leaving it as `com.twitter` will allow people to test with a learn/test heron easily.  This is probably the culprit for the weirdness I was experiencing the other day.

Reply:
Could we make a 0.17.8 versioned Helm release. Most Helm packages in the standard charts repo get published to this Google bucket store (https://console.cloud.google.com/storage/browser/kubernetes-charts-incubator). A repo like this would have versioned Helm artifacts. Then we could correct this code.